data/ecosystems/t/telos.toml

```diff
@@ -1,7 +1,24 @@
 # Ecosystem Level Information
 title = "Telos"
 
-sub_ecosystems = []
+sub_ecosystems = [
+    "OmniDex",
+    "Rubic",
+    "ApeSwap Finance",
+    "Zappy Finance",
+    "Sushi Swap",
+    "Elk Finance",
+    "PancakeSwap",
+    "Symbiosis Finance",
+    "Protofire",
+    "Actifit",
+    "API3",
+    "DIA",
```

Comment:
Most of these aren't dedicated Telos sub ecosystem.  We have a notion of primary and secondary ecosystems for the report, but this taxonomy only reflects primary ecosystems. 

Which of these are direct Telos subecos?